scripts/installer/glassfish-setup.sh

```diff
@@ -163,6 +170,12 @@ if [ -z "$DB_HOST" ]
   exit 1
 fi
 
+if [ -z "$DB_SERVICE" ]
```

Comment:
I don't think we want to add this requirement.  We should just not setup the extra datasource is DB_SERVICE isn't specified.

Reply:
Removed.